pdir/color.py

```diff
@@ -34,7 +34,7 @@ def __eq__(self, other: object) -> bool:  # type: ignore
         return self.color_code == other.color_code
 
     def __repr__(self):
-        return '\033[%sm%s\033[0m' % (self.color_code, 'color')
+        return '\033[{}m{}\033[0m'.format(self.color_code, 'color')
```

Comment:
use f-string for consistency?

Reply:
Interesting. I think pyupgrade and ruff didn't replace it with f-string because of `'color'`, which I assume was done this way for spellcheck purposes. I'll fix it.

Reply:
Done.